tests/fileTypeTests.py

```diff
@@ -110,7 +110,7 @@ def test_program_mixed(self):
         """Test mixed job with some rows failing."""
         jobId = self.jobIdDict["programMixed"]
         self.passed = self.run_test(
-            jobId, 200, "finished", 21060, 4, "complete", 118, True)
+            jobId, 200, "finished", 20998, 4, "complete", 118, True)
```

Comment:
Do these file size changes mean that we're reporting back short names now?  I'm not sure we want to be taking long names but reporting short names, that will be confusing.